en/user_guide/websites_creation.rst

```diff
@@ -0,0 +1,93 @@
+Website creation
+================
+
+Open Orchestra is multisite and allows to create several websites from the same Back Office application.
+
+Name
+----
+
+The field name contributes the name of the website and he allows to generate the identifier of the website.
```

Comment:
The field name is used for the website name and to generate the website identifier